site/profile/files/Get-PendingUpdate.ps1

```diff
@@ -0,0 +1,91 @@
+Function Get-PendingUpdate {
```

Comment:
Ideally, this file, and related files (pending.rb, wsus.pp) should be in their own module.  You can quickly create one and publish via github.